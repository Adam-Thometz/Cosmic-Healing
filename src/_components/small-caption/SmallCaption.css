.SmallCaption {
  font-family: 'IBM Plex Mono';
  letter-spacing: 2px;
  font-size: 16px;
}