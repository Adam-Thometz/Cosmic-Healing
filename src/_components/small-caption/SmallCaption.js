import React from 'react';

import './SmallCaption.css';

const SmallCaption = ({ text }) => (
  <small className='SmallCaption'>{text}</small>
);

export default SmallCaption;