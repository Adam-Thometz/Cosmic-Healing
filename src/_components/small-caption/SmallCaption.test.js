import React from "react";
import { render } from "@testing-library/react";
import SmallCaption from './SmallCaption';

describe('SmallCaption component', () => {
  it('renders without crashing', () => {
    render(<SmallCaption />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<SmallCaption />);
    expect(asFragment()).toMatchSnapshot();
  });
});