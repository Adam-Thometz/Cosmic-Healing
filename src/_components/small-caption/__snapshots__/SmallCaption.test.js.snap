// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallCaption component matches the snapshot 1`] = `
<DocumentFragment>
  <small
    class="SmallCaption"
  />
</DocumentFragment>
`;
